webv2/helpers/helpers.go

```diff
@@ -79,6 +79,24 @@ func createDatabase(ctx context.Context, uri string) error {
 				SchemaConversionObject JSON NOT NULL,
 				CreateTimestamp TIMESTAMP NOT NULL,
 			  ) PRIMARY KEY(VersionId)`,
+			  `CREATE TABLE JobExecutionData (
+				MigrationJobId STRING(50) NOT NULL,
+				SpannerDatabaseName STRING(50) NOT NULL,
+				AggMonitoringResources JSON,
+				IsShardedMigration BOOL NOT NULL,
+				CreateTimestamp TIMESTAMP NOT NULL,
+			  ) PRIMARY KEY(MigrationJobId)`,
+			  `CREATE TABLE ShardExecutionData (
+				MigrationJobId STRING(50) NOT NULL,
+				DataShardId STRING(50) NOT NULL,
```

Comment:
If we change any part of this execution pipeline, the design of the table would change. What is the thought process of handling such scenarios.

Reply:
My thought process is evolve the schema as we have a line of sight on what different types of jobs would look like. 
Short of defining everything in a NoSQL like fashion and storing job related information as a JSON, it will be hard to design a schema which fits very varied possible job designs. Even storing it in JSON would mean shifting the complexity to the application rather than solving the problem itself.